test/config.js

```diff
@@ -75,6 +75,24 @@ test('no secretToken', function (t) {
   })
 })
 
+test('has apiKey', function (t) {
```

Comment:
I see here we've created a mocked APMServer and use that mock to ensure the configuration value end up in the Authorization header.  This test matches the pattern used for other values. 👍